### PR TITLE
Continuous play / title in playbar fragment

### PR DESCRIPTION
**Pull request by theJenix**
This was foundational, before I could tackle transfer song.  I also fixed the pause/resume functionality so it doesnt restart the song, and added a broadcast intent for the current song, and placed the title in the playbar fragment.  We can further style it however we like.

### CODE REVIEW

File: SoundStream/src/com/lastcrusade/soundstream/audio/RemoteAudioPlayer.java [hunk 1]
```diff
@@ -57,9 +73,11 @@ private void registerReceivers() {
 			public void onReceiveAction(Context context, Intent intent) {
 				playing = intent.getBooleanExtra(MessagingService.EXTRA_IS_PLAYING, false);
 				if(playing) {
+				    paused = false;
```
Comment (reidbaker) [inline SoundStream/src/com/lastcrusade/soundstream/audio/RemoteAudioPlayer.java:76]: You can fix later but there is weird spacing here

**Comment by reidbaker** [comment]
:thumbsup: Next person that likes it pull this in.

File: SoundStream/src/com/lastcrusade/soundstream/audio/SingleFileAudioPlayer.java [hunk 2]
```diff
@@ -43,7 +45,7 @@ public void setSongByPath(String filePath) {
 
     @Override
     public boolean isPlaying() {
-        return player.isPlaying();
+        return player.isPlaying() && !paused;
```
Comment (ejohnson44) [inline SoundStream/src/com/lastcrusade/soundstream/audio/SingleFileAudioPlayer.java:48]: Is there a reason that we are looking at player's playing, but our own pause? Later it looks like player has a paused state as well, so should we be using both of those here?

**Comment by dgreenhalgh** [comment]
:shipit:

File: SoundStream/src/com/lastcrusade/soundstream/audio/SingleFileAudioPlayer.java (hunk 2 above)
Comment (reidbaker) [inline SoundStream/src/com/lastcrusade/soundstream/audio/SingleFileAudioPlayer.java:48]: Was this addressed ?

**Pull request merged**